docs/storage_snippets.py

```diff
@@ -39,7 +39,8 @@ def storage_get_started(client, to_delete):
     bucket = client.get_bucket('bucket-id-here')
     # Then do other things...
     blob = bucket.get_blob('/remote/path/to/file.txt')
-    print(blob.download_as_string())
+    
+    (blob.download_as_string())
```

Comment:
I'm guessing I was trying to get the search to work on the github UI editor. :/ *facepalm*